dotnet-efcore/RecordsAsModelClasses/RecordsAsModelClasses/Core/DTOs/CarDto.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace RecordsAsModelClasses.Core.DTOs;
+
+public record CarDto(int Id, string Make, string Model, int Year);
```

Comment:
After you read the article review from senior editors, let's keep this CarDto only for returned value and we should create another Dto called UpsertCarDto that is similar to this but doesn't have an Id, and we should use this new on in Action methods that Add or update a Car since we are not needing the Id parameter for that. Something like the following : 
``
public record UpsertCardDto(string Make, string Model, int Year);
``